### ISSUE TITLE
Notice of Intellectual Property Infringement and NDA Violation

### ISSUE THREAD

**Issue by kool79** [AleynaSarisinn/AutomationTask-Insider#1]
Hi, @AleynaSarisinn 
We have been informed about the unauthorized use and publication of code (in the AutomationTask-Insider repository)  that belongs to Astound Digital, to which you had access during your employment with the company. Please be reminded that this code is the intellectual property of Astound Digital and is subject to the terms of the Non-Disclosure Agreement (NDA) that you signed.

We strongly demand that you immediately cease any use or distribution of this code, and delete it **from your local computer as well as from any online accounts or repositories** where it may be stored or shared.  If any further violations occur, Astound Digital will take appropriate actions, including, but not limited to, legal measures for breach of intellectual property rights and the NDA.

**Comment by AleynaSarisinn** [comment]
Hello @kool79,
Yesterday, I encountered an issue where I could see all the previous commits we had made in the project. I tried to fix it and checked all the relevant settings, such as the account and email configurations in the IDE. However, whenever I push a new commit, I still see the previous Astound files included in my commit. I have resolved the issue, and now my commits no longer contain any files related to your side.

Here is the previous screenshot for reference:
https://prnt.sc/xjOCb7BUe9nV

To summarize, the repository AutomationTask-Insider does not contain any fields related to the Astound code, and I do not have any copies on my end. I will check the repository again immediately, and if it does contain any data related to Astound, I will delete it.

**Comment by AleynaSarisinn** [comment]
@kool79 
I checked all the files in the AutomationTask-Insider repository, but I don't see any Astound files. If you could specify which files you're referring to, I can check them immediately.

**Comment by kool79** [comment]
> Yesterday, I encountered an issue where I could see all the previous commits we had made in the project.

It is not an issue,You actually published all source code into AutomationTaskInsider project (with a similar name, like current one) and made the sources publicly available. Please remove **the whole folder with the Astound project** from your local PC.
It is how git works. Even if you don't see files in your local working copy, all previous history of the project is saved. 

> I checked all the files in the AutomationTask-Insider repository, but I don't see any Astound files. 

Sorry, I made mistake in my initial message about project name. The project, where you shared our code, had the name  AutomationTaskInsider, not the AutomationTask-Insider
Link to the commit: 
[https://github.com/AleynaSarisinn/AutomationTaskInsider/blob/d82230679e23e44878318f6c0b52a0ff2f90bc2b](https://github.com/AleynaSarisinn/AutomationTaskInsider/blob/d82230679e23e44878318f6c0b52a0ff2f90bc2b/shared/ext/test-status-filter/credentials/test-automation-sheets-api.json)
  Please remove this repo as well from you github account (if you did not do that already)

**Comment by AleynaSarisinn** [comment]
The project is not located on my computer. While pushing the AutomationTaskInsider repository, I noticed 2 or 3 commit messages related to Astound in the actual code that I intended to send.

These commits were not visible in my commit history; they only appeared on GitHub.
The repository you mentioned was only created to check for the existence of the issue, and as you may have seen, I deleted it immediately yesterday.


You can be assured that I am fully aware that Astound code must not be shared or replicated, and I understand that doing so would constitute a violation.
